app/models/caches/plan_details.rb

```diff
@@ -39,12 +44,12 @@ def self.load_record_cache!
             {
               :start_on => pt.start_on,
               :end_on => pt.end_on,
-              :cost => pt.cost
+              :cost => pt.cost,
+              :rating_area => pt.rating_area
```

Comment:
The goal here was actually 2 fold - introduce a new lookup method and don't change the existing interface.  This changes the existing interface.